src/Umbraco.Web.UI.Client/src/common/directives/components/contenttype/umbcontenttypeproperty.component.js

```diff
@@ -31,6 +32,12 @@
       }
     }
 
+    function clickComposition(documentTypeId) {
+      if (vm.onClickComposition) {
+        vm.onClickComposition({ documentTypeId: documentTypeId });
```

Comment:
Would `documentTypeId` be a better parameter name to match component?

Reply:
For now I took the same naming as the ones we have in https://github.com/umbraco/Umbraco-CMS/blob/5bfab13dc5a268714aad2426a2b68ab5561a6407/src/Umbraco.Web.UI.Client/src/common/directives/components/contenttype/umbcontenttypetab.component.js#L62 & https://github.com/umbraco/Umbraco-CMS/blob/5bfab13dc5a268714aad2426a2b68ab5561a6407/src/Umbraco.Web.UI.Client/src/common/directives/components/contenttype/umbcontenttypegroup.component.js#L48.

I can change this one, but then I think it would be smart to also change the other ones.

Reply:
Yes, I think the argument name shouldn't matter, but fortunately the function is called `clickComposition` and not `clickDocumentTypeComposition` 😁